django/contrib/gis/db/models/fields.py

```diff
@@ -243,6 +243,12 @@ def deconstruct(self):
             kwargs['dim'] = self.dim
         if self.geography is not False:
             kwargs['geography'] = self.geography
+
+        # Oracle-specific attributes for `USER_SDO_GEOM_METADATA`
```

Comment:
I'm not sure if we need this comment (and blank line), because the same is already in `__init__()` :thinking:.